src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MemberNameCheck.java

```diff
@@ -56,16 +56,74 @@
  * <pre>
  * &lt;module name="MemberName"/&gt;
  * </pre>
```

Comment:
please put example of `applyToXxxxxx` to configs and examples.
for example `applyToPublic` and `applyToProtected ` to first example and `applyToPackage ` and `applyToPrivate ` to second example.

users love examples so mush that they skip reading of documentation at all. I think you noticed the same with you.

Reply:
I have made some little changes in second example. However, I think in the first example, users could see how the default config works. In the third example, I put the  applyToPublic & applyToPrivate which could combine with the applyToPackage & applyToProtected in the second example.

Reply:
Done.